README.md

```diff
@@ -0,0 +1,28 @@
+# chaski
+chaski mission is to help delivering upstream code to downstream build system.
+
+## Creating a downstream build
+
+Before running chaski, you will need to: 
+- clone discovery-server from `dist-git`
+- switch to an appropriate branch (or create a new one prefixed by `private-` if you are experimenting)
+- create/edit a file named `sources-version.yaml`. It should look like this
```

Comment:
Maybe here you could create an example.yaml and reference it here

Reply:
:thinking: I think I didn't got what you mean because the next lines are doing exactly that.

Reply:
I mean really creating an example.yaml. It is perfectly fine like this, it's only worth creating if config is gonna get complex at some point.

Reply:
if it were complex, sure. but that's not the goal with this file - it is meant as just a small complement to `container.yaml`, adding this extra bit of information (commitsh instead of an actual sha).